WapoJS/src/host_functions/https_listen.rs

```diff
@@ -223,13 +223,6 @@ async fn serve_connection(weak_service: ServiceWeakRef, id: u64, connection: Tcp
             service_fn(move |req| {
                 let weak_service = weak_service.clone();
                 async move {
-                    if req.method() == hyper::Method::OPTIONS {
```

Comment:
What would happen if the script doesn't handle the OPTIONS?

Reply:
Allows guest scripts handling the CORS preflight requests.

Reply:
I mean if the guest script ignored OPTIONS (by default), and an OPTIONS request comes, would it returns error if we removed the handling in the host?
If so, do you think we still need a default 200 response if the script ignored OPTIONS?

Reply:
> I think it’s better to do nothing. The default behavior could cause confusion for developers when debugging their code.

make sense